docs/plain-bundle-spec.md

```diff
@@ -0,0 +1,138 @@
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for a [Bundle](/README.md#Bundle)
+resource.
+
+A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory. A plain
+bundle can consist of a container image, a directory in a git repository, or any other content source that
+the [plain bundle provisioner](/internal/provisioner/plain/README.md) supports.
+
+The currently implemented plain bundle format is the `plain+v0` format. The name of the bundle format, `plain+v0`
+combines the type of bundle (plain) with the current schema version (v0).
+The [plain bundle provisioner](/internal/provisioner/plain/README.md) is able to source `plain+v0` bundles and install
+them onto a Kubernetes cluster.
+
+> Note: the plain+v0 bundle format is at schema version v0, which means it's an experimental format that is subject
+> to change.
+
+Supported source types for a plain bundle currently include the following:
+
+* A container image
+* A directory in a git repository
+
+Additional source types, such as a local volume or a generic URI-based resource, are on the roadmap. These source types
+all present the same content, a directory containing static Kubernetes YAML manifests, in a different ways.
+
+## Example
+
+For example, below is a minimal example of a Dockerfile that builds a `plain+v0` image-based bundle from a directory
+containing static Kubernetes manifests.
+
+```dockerfile
+FROM scratch
+COPY /manifests /manifests
+```
+
+where the given `manifests` directory contains the Kubernetes resources required to deploy an application, for example:
+
+```
+manifests
+├── namespace.yaml
+├── cluster_role.yaml
+├── role.yaml
+├── serviceaccount.yaml
+├── cluster_role_binding.yaml
+├── role_binding.yaml
+└── deployment.yaml
+```
+
+For an example of a directory backing a git-based bundle,
+see [the combo manifests directory](https://github.com/operator-framework/combo/tree/main/manifests). Any directory that
+contains only static Kubernetes manifests checked into a git repository accessible via a remote URL can be considered a
+plain bundle and sourced by the plain provisioner.
+
+> Note: there must be at least one resource in the manifests directory in order for the bundle to be a valid
+> plain+v0 bundle.
+
+## Technical Details
+
+* The static manifests must be located in the root-level /manifests directory in a bundle image for the bundle to be a
+  valid `plain+v0` bundle that the provisioner can unpack. A plain bundle image without a /manifests directory is
+  invalid and will not be successfully unpacked onto the cluster. For a git-based plain bundle, this limitation does not
+  exist, and the manifests can be in any directory in the repository. The manifest directory is assumed to be
+  ./manifests in a git-based plain bundle but can be overwritten.
```

Comment:
Should we define some terms somewhere?
- `bundle` is a collection of files that define content to be deployed to a cluster.
- `bundle image` is a container image that contains a bundle (any reason the bundle has to be at the image root? we could augment the `image` source type to allow specifying a subdirectory in the image, defaulting to `/manifests`, just like we're doing for bundle repos)
- `bundle git repo` is a git repo that contains a bundle (where the bundle can be anywhere within the repo)

Reply:
Yeah I think adding a common terminology section might help make this easier to reason about over time?